CHANGELOG.md

```diff
@@ -2,6 +2,9 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
+
```

Comment:
Nit: we tend to add something like `(None)` after HEAD to make it clear that's there's nothing pending